bug: pvc size no verify

I found the following error when PVC creation failed.

It looks like this needs to be fixed.

```bash
2023-07-21T10:57:43Z	INFO	Observed a panic in reconciler: cannot parse '': quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$'	{"controller": "infrablockspace", "controllerGroup": "infrablockspace.net", "controllerKind": "InfraBlockSpace", "InfraBlockSpace": {"name":"infrablockspace-sample","namespace":"bc-core"}, "namespace": "bc-core", "name": "infrablockspace-sample", "reconcileID": "e05ec30d-5a2e-4f6d-bf7d-6e209d00f537"}
panic: cannot parse '': quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$' [recovered]
	panic: cannot parse '': quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$'
```

--- a/controllers/infrablockspace_controller.go
+++ b/controllers/infrablockspace_controller.go
@@ -219,6 +219,9 @@ func (r *InfraBlockSpaceReconciler) validateKey(key chain.Key) error {
 }
 
 func (r *InfraBlockSpaceReconciler) createChainPVC(ctx context.Context, name, namespace, size, scName string) (ctrl.Result, error) {
+	if size == "" {
+		size = "100Gi"
+	}
 	pvc := chain.CreateChainPVC(name, namespace, size, scName)
 	if err := r.Create(ctx, pvc); err != nil {
 		logger.Error(err)
@@ -232,6 +235,9 @@ func (r *InfraBlockSpaceReconciler) createChainPVC(ctx context.Context, name, na
 	return ctrl.Result{Requeue: true}, nil
 }
 func (r *InfraBlockSpaceReconciler) updateChainPVC(ctx context.Context, name, namespace, size string) (ctrl.Result, error) {
+	if size == "" {
+		return ctrl.Result{}, errors.New("size is required")
+	}
 	foundPVC := &corev1.PersistentVolumeClaim{}
 	if err := r.Get(ctx, types.NamespacedName{Name: name, Namespace: namespace}, foundPVC); err != nil {
 		logger.Error(err)

